src/random.cpp

```diff
@@ -175,7 +175,7 @@ static void RandAddSeedPerfmon()
 #endif
 }
 
-#ifndef WIN32
+#if !defined(WIN32) && !(defined(HAVE_GETENTROPY) && defined(__OpenBSD__))
```

Comment:
I think we need to encode the full `GetOSRand` elif tree up to that point. i.e.:
`#if !defined(WIN32) && (defined(HAVE_SYS_GETRANDOM) || !(defined(HAVE_GETENTROPY) && defined(__OpenBSD__))`

Reply:
Updated. Please re-review :-)